diff --git a/imagex_installkit.info b/imagex_installkit.info
index ea5c38e..0cad912 100644
--- a/imagex_installkit.info
+++ b/imagex_installkit.info
@@ -77,7 +77,6 @@ dependencies[] = views_bulk_operations
 ; ImageX modules and features to be installed and enabled.
 dependencies[] = imagex
 dependencies[] = imagex_bean
-dependencies[] = imagex_bean_permissions
 dependencies[] = imagex_common_fields
 dependencies[] = imagex_files
 dependencies[] = imagex_media
@@ -86,10 +85,7 @@ dependencies[] = imagex_page
 dependencies[] = imagex_page_workflow
 dependencies[] = imagex_panels
 dependencies[] = imagex_user_roles
-dependencies[] = imagex_user_permissions
 dependencies[] = imagex_vocabulary_tags
 dependencies[] = imagex_vocabulary_tags_field
-dependencies[] = imagex_vocabulary_tags_permissions
 dependencies[] = imagex_workflow
 dependencies[] = imagex_wysiwyg_profiles
-dependencies[] = imagex_wysiwyg_profiles_permissions
